ipi_tests/regression_tests/slc/input.xml

```diff
@@ -0,0 +1,50 @@
+<simulation verbosity="high">
+  <ffsocket mode='unix' name='wat1beads' pbc='true'>
+    <port> 33330 </port>
```

Comment:
is there any reason to add a port to a unix socket here? leaving it in tends to create confusion to users sometimes, even though it does not harm in any way. maybe a good option would be using inet/localhost also here just to make it clear how it would work with real different ports? not super important.

Reply:
are you suggesting changing everything to inet/localhost? or to add at least one regression test where inet/localhost is used?

Reply:
I think there are already tests where inet comm is used. In slc it is useful just to show the setup using different ports. So yes, I would tend to try inet/localhost in this case. But this should not be a reason not to merge this ASAP. @venkatkapil24 and @ceriottm -- fine?

Reply:
No problem, I'll change this example to use inet/localhost